lib/ansible/modules/notification/slack.py

```diff
@@ -55,6 +55,10 @@
   channel:
     description:
       - Channel to send the message to. If absent, the message goes to the channel selected for the I(token).
+  thread_ts:
```

Comment:
From a UI perspective, `thread_ts` is an obtuse name. Yes, it's accurate to the API, but maybe `thread_id` would be a better name. It could still be `thread_ts` internally.

Reply:
Also, thanks for adding this. Very nice feature!